runner/package.json

```
@@ -7,7 +7,8 @@
     "node": "18.17"
   },
   "scripts": {
-    "build": "rm -rf ./dist && tsc",
+    "build": "rm -rf ./dist && npm run codegen && tsc",
+    "codegen": "npx proto-loader-gen-types --longs=String --enums=String --defaults --oneofs --grpcLib=@grpc/grpc-js --outDir=src/generated/ protos/runner.proto",
```

```suggestion
    "codegen": "proto-loader-gen-types --longs=String --enums=String --defaults --oneofs --grpcLib=@grpc/grpc-js --outDir=src/generated/ protos/runner.proto",
```
`npx` is only needed if you call this from a standalone cli. `npm` scripts will automatically have access to `node_modules` `bin`